### PR TITLE
Add left and right screen layout to web config (#106)

### PR DESCRIPTION
**Pull request by deeebug**
As per the requirements in #106, the following changes expose options for the button layouts to be altered with the web configurator.

Please review and merge. Thank you.

### CODE REVIEW

File: www/src/Pages/DisplayConfig.js [hunk 1]
```diff
@@ -17,6 +17,36 @@ const I2C_BLOCKS = [
...
+	{ label: 'Dancepad', value: 6 },        // BUTTON_LAYOUT_DANCEPADA
+	{ label: 'Twinstick', value: 7 },       // BUTTON_LAYOUT_TWINSTICKA
+	{ label: 'Blank', value: 8 },           // BUTTON_LAYOUT_BLANKA
+	{ label: 'VLX', value: 9 }              // BUTTON_LAYOUT_VLXA
+];
+
+const BUTTON_LAYOUT_RIGHT = [
+	{ label: 'Arcade', value: 0 },			 // BUTTON_LAYOUT_ARCADE
+	{ label: 'Stickless', value: 1 },        // BUTTON_LAYOUT_STICKLESSB
+	{ label: 'Buttons Anbled', value: 2 },   // BUTTON_LAYOUT_BUTTONS_ANGLEDB
```
Comment (arntsonl) [inline www/src/Pages/DisplayConfig.js:36]: Changed "Anbled" to "Angled"

**Comment by arntsonl** [comment]
Added a typo in the comments (let me know if you don't see it), once that's fixed we'll merge in this PR.

**Comment by deeebug** [comment]
Fixed, please check and merge.

**Comment by arntsonl** [comment]
Looks great, thanks for adding this!

**Pull request merged**